theme/styles/abstracts/_variables.scss

```diff
@@ -66,9 +66,9 @@ $header: (
   gap: 0,
   layout-breakpoint: 1625px,
   breakpoint-for-search-bar: 850px,
-  nav-link-padding-lateral: 0.8em,
-  nav-width-desktop: 300px,
-  nav-width-mobile: 225px,
+  nav-link-padding-lateral: 1em,
+  nav-width-desktop: 600px,
+  nav-width-mobile: 300px,
```

Comment:
updating sass vars to match updated design.  One design departure is Jose showed the 1st level mobile nav significantly narrower than the 2nd level.  It looked more balanced to me to keep the 1st and 2nd level the same width on most screen sizes, so that's what I did.